src/utils/translator.js

```diff
@@ -0,0 +1,92 @@
+"use strict";
+
+class Translator {
+  constructor(options = {}) {
+    this._options = Object.assign({}, this.defaultConfig, options);
+    this._lang = this.getLanguage();
+    this._elements = document.querySelectorAll("[i18n]");
+  }
```

Comment:
You could call `this.load(options.lang)` right away in the constructor below these other lines. Or better yet, check to see if it's not the default language first:

```js
if (this._lang !== this._options.defaultLang) {
  this.load(this._options.lang)
}
```